src/helm/proxy/clients/lit_gpt_client.py

```diff
@@ -0,0 +1,165 @@
+import json
+import logging
+import time
+from pathlib import Path
+from threading import Lock
+from typing import Literal, Optional
+
+import lightning as L
+import torch
+from lightning.fabric.strategies import FSDPStrategy
+from lit_gpt import GPT, Config, Tokenizer
+from lit_gpt.model import Block
+from lit_gpt.utils import check_valid_checkpoint_dir, lazy_load, quantization
```

Comment:
I have added lit-gpt as requirement. Since lit-gpt doesn't come as a pip package so I have added it as `lit-gpt @ git+https://github.com/Lightning-AI/lit-gpt@main`. It might not be added to pypi in this form though.